### DISCUSSION TITLE
The release 1.3.1 foxy not supported action , actionGloal

### DISCUSSION THREAD

**Discussion by RafaaZahra** [RobotWebTools/rosbridge_suite#902] [General]
Im working with jeston nano ros2 foxy , but only rosbridge_suite 1.3.1 supported in foxy, Is there an update to version 1.3.2 for foxy?

**Comment by sea-bass** [comment]
ROS 2 Foxy is end of life as of last year, so I don't think there will be more binary releases to that.

If you want, you can try building the `ros2` branch of this repo from source on your Jetson, with the possibility that it may not work.

**Reply by RafaaZahra** [reply]
i try to install the new version for ros2 1.3.2 but doesn't work , now i will search for any tutorial how setup ros2 humble in jeston nano beacuase jeston nano support  only ros2 foxy , thanks for the response @sea-bass
